### ISSUE TITLE
artman fails to generate go gapics

### ISSUE THREAD

**Issue by jadekler** [googleapis/artman#469]
`googleapis-artman-0.14.0`

```
+ artman --config google/iam/artman_iam_admin.yaml generate go_gapic
Traceback (most recent call last):
  File "/Users/deklerk/workspace/go/src/github.com/googleapis/googleapis/env/bin/artman", line 7, in <module>
    from artman.cli.main import main
  File "/Users/deklerk/workspace/go/src/github.com/googleapis/googleapis/env/lib/python3.6/site-packages/artman/cli/__init__.py", line 17, in <module>
    from artman.cli import configure
  File "/Users/deklerk/workspace/go/src/github.com/googleapis/googleapis/env/lib/python3.6/site-packages/artman/cli/configure.py", line 30, in <module>
    from artman.config.proto.user_config_pb2 import UserConfig, LocalConfig, GitHubConfig
  File "/Users/deklerk/workspace/go/src/github.com/googleapis/googleapis/env/lib/python3.6/site-packages/artman/config/proto/user_config_pb2.py", line 22, in <module>
    serialized_pb=_b('\n\x11user_config.proto\x12\x11googleapis.artman\"l\n\nUserConfig\x12-\n\x05local\x18\x01 \x01(\x0b\x32\x1e.googleapis.artman.LocalConfig\x12/\n\x06github\x18\x02 \x01(\x0b\x32\x1f.googleapis.artman.GitHubConfig\"\x1e\n\x0bLocalConfig\x12\x0f\n\x07toolkit\x18\x01 \x01(\t\"/\n\x0cGitHubConfig\x12\x10\n\x08username\x18\x01 \x01(\t\x12\r\n\x05token\x18\x02 \x01(\tb\x06proto3')
TypeError: __new__() got an unexpected keyword argument 'serialized_options'
```

**Comment by garrettjonesgoogle** [comment]
This looks like @andreamlin 's change in https://github.com/googleapis/artman/pull/462 to regenerate from user_config.proto

**Comment by lukesneeringer** [comment]
@jadekler If you update `protobuf` in your virtualenv to 3.6.0, it should fix it.

Filed #470 to fix it for everyone.

**Comment by alexander-fenster** [comment]
OK how did this pass smoke tests then?

**Comment by alexander-fenster** [comment]
Or did it just fetch the latest protobuf when it was installed in CI and pass because of that?

**Comment by alexander-fenster** [comment]
@jadekler I have just released v0.14.1 with the fix from @lukesneeringer. Please check it and let us know if it works for you.

**Comment by jadekler** [comment]
Thank you! It works for me.
